src/Tasks/Microsoft.NET.Build.Tasks/IPackageResolver.cs

```diff
@@ -8,5 +8,6 @@ namespace Microsoft.NET.Build.Tasks
     public interface IPackageResolver
     {
         string GetPackageDirectory(string packageId, NuGetVersion version);
+        string GetPackageDirectory(string packageId, NuGetVersion version, out string packageRoot);
```

Comment:
@nguerrera correct me if i am wrong- IPackageResolver is more like the interface sdk consumes, kind of like PAL that abstracts Nuget package management. So if the interface changes it is unlikely to ever break our consumers.

As of now this interface is only implemented by sdk